hcs-sxc-java-examples/hcs-sxc-java-settlement-demo/src/main/java/com/hedera/hcsapp/integration/HCSIntegration.java

```diff
@@ -244,8 +245,10 @@ public void processHCSMessage(HCSResponse hcsResponse) {
         
                                 settlementRepository.save(settlement);
         
-                                new OutboundHCSMessage(appData.getHCSCore()).overrideEncryptedMessages(false)
-                                        .overrideMessageSignature(false).sendMessage(appData.getTopicIndex(), newSettlementBPM.toByteArray());
+                                OutboundHCSMessage o =  new OutboundHCSMessage(appData.getHCSCore());
+                                        o.overrideEncryptedMessages(true);
+                                        o.overrideMessageSignature(false);
```

Comment:
Message signature should still be optional on unencrypted messages, but HAS to be included on encrypted messages.

Reply:
See issue #373